k8s/namespaces/admin/traefik-private/patches/demo-00.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   namespace: admin
 spec:
   values:
-    loadBalancerIP: 10.51.15.221
\ No newline at end of file
+    loadBalancerIP: 10.50.79.221
```

Comment:
Hi Darren, can you review these - so k8s/ folder is flux v1 and this is on v2 for demo now - but it should be quick to find those v2 files instead

Reply:
undone the previous change for flux v1 and put changes into the flux v2 folder only.